app/ConsoleWriter.php

```diff
@@ -116,8 +116,30 @@ public function verbose()
         return $this;
     }
 
-    public function shouldWriteLine()
+    private function labeledLine(string $label, string $message, string $labelFormat = 'fg=default;bg=default', int $indent = 0): void
     {
-        return $this->getVerbosity() > SymfonyStyle::VERBOSITY_NORMAL;
+        $indentString = str_repeat(' ', $indent);
+        $this->isDecorated()
+            ? parent::text("{$indentString}<{$labelFormat}> {$label} </> {$message}")
+            : parent::text("{$indentString}[{$label}] {$message}");
+    }
+
+    private function shouldWriteLine($ignoreVerbosity = true)
```

Comment:
This is a weird method... in any context that we're passing `true`, we don't even need this method, right? 

Could we replace it with a `resetVerbosity` method that we call at the end of each method, maybe? That way only methods that might actually have a `false` return from this method actually need to call it.

Reply:
Done.

I have revamped and simplified the code that outputs to console.